File: JavaScript_Courses/JavaScript_Basics/JS_BP_2022_10/03_Conditional_Statements_Advanced_Exercise/06_Operations_Between_Numbers.js
function operationsBetweenNumbers(input) {  // ["10", "12", "+"]
    let num1 = Number(input[0]);
    let num2 = Number(input[1]);
    let operator = input[2];

    let result = 0;
    let resultAsString = "";

    if (operator == "+") {
        result = num1 + num2;
        let evenOrOdd = "";

        if (result % 2 == 0) {
            evenOrOdd = "even";
        } else {
            evenOrOdd = "odd";
        }

        resultAsString = `${num1} ${operator} ${num2} = ${result} - ${evenOrOdd}`;

    } else if (operator == "-") {
        result = num1 - num2;
        let evenOrOdd;

        if (result % 2 == 0) {
            evenOrOdd = "even";
        } else {
            evenOrOdd = "odd";
        }

        resultAsString = `${num1} ${operator} ${num2} = ${result} - ${evenOrOdd}`;

    } else if (operator == "*") {
        result = num1 * num2;
        let evenOrOdd;

        if (result % 2 == 0) {
            evenOrOdd = "even";
        } else {
            evenOrOdd = "odd";
        }

        resultAsString = `${num1} ${operator} ${num2} = ${result} - ${evenOrOdd}`;
        
    } else if (operator == "/") {
        if (num2 == 0) {
            resultAsString = `Cannot divide ${num1} by zero`;
        } else {
            result = num1 / num2;
            resultAsString = `${num1} ${operator} ${num2} = ${result.toFixed(2)}`;
        }

    } else if (operator == "%") {
        if (num2 == 0) {
            resultAsString = `Cannot divide ${num1} by zero`;
        } else {
            result = num1 % num2;
            resultAsString = `${num1} ${operator} ${num2} = ${result}`;
        }
        
    }

    console.log(resultAsString);
}

// operationsBetweenNumbers(["10", "12", "+"]);

File: Python_Courses/Python_Basics/Preparation/03_Conditional_Statements_Advanced_Exercise/06_Operations_Between_Numbers.py
n1 = int(input())
n2 = int(input())
math_operator = input()

result = 0
even_or_odd = ""
if math_operator == "+":
    result = n1 + n2
    if result % 2 == 0:
        even_or_odd = "even"
    else:
        even_or_odd = "odd"
    print(f"{n1} {math_operator} {n2} = {result} - {even_or_odd}")
elif math_operator == "-":
    result = n1 - n2
    if result % 2 == 0:
        even_or_odd = "even"
    else:
        even_or_odd = "odd"
    print(f"{n1} {math_operator} {n2} = {result} - {even_or_odd}")
elif math_operator == "*":
    result = n1 * n2
    if result % 2 == 0:
        even_or_odd = "even"
    else:
        even_or_odd = "odd"
    print(f"{n1} {math_operator} {n2} = {result} - {even_or_odd}")
elif math_operator == "/":
    if n2 == 0:
        print(f"Cannot divide {n1} by zero")
    else:
        result = n1 / n2
        print(f"{n1} / {n2} = {result :.2f}")
elif math_operator == "%":
    if n2 == 0:
        print(f"Cannot divide {n1} by zero")
    else:
        result = n1 % n2
        print(f"{n1} % {n2} = {result}")
